src/model/scripts/llama_thai_tokenizer/README.md

```diff
@@ -1,25 +1,47 @@
-# Tokenizer Experiment
+# OpenThaiGPT Merged Tokenizer Pipeline
 
-merge LLaMa Tokenizer with [wangchanberta](https://huggingface.co/airesearch/wangchanberta-base-att-spm-uncased/tree/main) Tokenizer by extent vocabulary following [working example](https://github.com/ymcui/Chinese-LLaMA-Alpaca/blob/main/README_EN.md) 
+we plan to use [Pretrained LLaMA Model](https://ai.meta.com/blog/large-language-model-llama-meta-ai/) as a base model for finetuning but LLaMa Tokenizer (BPE Tokenizer) have less Thai vocabulary, this pipeline intent to extent vocabulary of LlaMA tokenizer, using merge method following [Chinese-LLaMA-Alpaca](https://github.com/ymcui/Chinese-LLaMA-Alpaca/blob/main/README_EN.md)
```

Comment:
`this pipeline intent to` -> `This pipeline intend to`

Reply:
done